demo/demo.js

```diff
@@ -1,20 +1,13 @@
 hljs.debugMode();
 hljs.highlightAll();
 
-const addEventListeners = (element, listener) => {
-  element.addEventListener("click", listener);
-  element.addEventListener("keydown", (event) => {
-    if (event.key === "Enter") {
-      listener(event);
-    }
-  });
-};
+document.querySelectorAll(".categories a").forEach((category) => {
+  category.addEventListener("click", (event) => {
+    event.preventDefault();
 
-document.querySelectorAll(".categories > li").forEach((category) => {
-  addEventListeners(category, (event) => {
     const current = document.querySelector(".categories .current");
-    const currentCategory = current.dataset.category;
-    const nextCategory = event.target.dataset.category;
+    const currentCategory = current.name;
+    const nextCategory = event.target.name;
```

Comment:
Using dataset seems entirely appropriate here to name the type of data vs using the generic HTML `name`... was there a reason for this change?

Reply:
No, just copying from my testing. :D